src/domain/event/EditEventPage.tsx

```diff
@@ -201,7 +202,7 @@ const EditEventPage: React.FC = () => {
       <LoadingSpinner isLoading={loading}>
         {!!eventData ? (
           <>
-            {isEditableEvent(eventData) ? (
+            {isEditableEvent ? (
```

Comment:
I would still use isEditableEvent(eventData) function, but change the internal logic. Just to keep maintaining little bit easier. Now ```const isEditableEvent =
    eventData?.event?.publicationStatus === PUBLICATION_STATUS.DRAFT; ``` is duplicated